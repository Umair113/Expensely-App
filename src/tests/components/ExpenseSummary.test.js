import React from 'react';
import {shallow} from 'enzyme';
import {ExpensesSummary} from '../../Components/ExpenseSummary';

test('should correctly render ExpensesSummary with 1 expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={235}/>);
    expect(wrapper).toMatchSnapshot();
});

test('should correctly render ExpensesSummary with 1 expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={23523123}/>);
    expect(wrapper).toMatchSnapshot();
});