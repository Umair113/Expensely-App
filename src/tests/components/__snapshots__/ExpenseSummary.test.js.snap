// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render ExpensesSummary with 1 expense 1`] = `
<div>
  <h2>
    Viewing 
    1
     
    expense
     
  </h2>
   
  <h3>
    Total: 
    $0.00
  </h3>
</div>
`;

exports[`should correctly render ExpensesSummary with 1 expense 2`] = `
<div>
  <h2>
    Viewing 
    23
     
    expenses
     
  </h2>
   
  <h3>
    Total: 
    $0.00
  </h3>
</div>
`;
